src/main/java/dev/enjarai/trickster/spell/trick/Tricks.java

```diff
@@ -324,6 +324,7 @@ public RegistryEntry.Reference<Trick> add(RegistryKey<Trick> key, Trick value, R
     public static final GetRedstonePowerTrick GET_REDSTONE_POWER = register("get_redstone_power", new GetRedstonePowerTrick());
     public static final HeatTrick HEAT = register("heat", new HeatTrick());
     public static final CoolTrick COOL = register("cool", new CoolTrick());
+    public static final WeatherTrick WEATHER = register("weather", new WeatherTrick());
```

Comment:
I'd actually prefer it be called "erosion". It also makes sense for copper imo.

Reply:
Hello. Resident bikeshedder here. "Cool" and "heat" are dissimilar to "erosion." Erosion is a *thing*, not an *action*. Did you mean "erode?"

Seriously though, it's not like we'll be adding anything related to the weather in the sky anyhow, and copper is "exposed" and "weathered."

Granted all this is merely a choice of internal flavour, the docs won't have a name with either.